_posts/2021-01-08-diablo-2-ocr.md

```diff
@@ -0,0 +1,234 @@
+---
+title:  Diablo 2 OCR
+date: 2021-01-08 00:00:00 +0800
+author: eric
+---
+
+If you can believe it: a few neat tools I made to help me _get better items
+in the video game Diablo 2_ are the bedrock of my programming career.
+
+"Oh it'd be so cool if I could make a program that automatically
+picked up good items in this game I am playing..!"
+
+This post is about the Optical Character Recognition (OCR) tool I made to 
+pickup good items and encouraging kids to pursue weird passions.
+
+<!--more-->
```

Comment:
Looks like your `<!--more-->` is a little too far down.
![image](https://user-images.githubusercontent.com/16579931/103970668-a5a6b700-511d-11eb-94d0-6c91019bab2e.png)

Reply:
I think I made this have varying height and we changed that? should we bring back varying height/redesign the listing page to avoid this in the future?